src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldLikeEventSymbol.cs

```diff
@@ -113,9 +113,11 @@ internal SourceFieldLikeEventSymbol(SourceMemberContainerTypeSymbol containingTy
                 }
             }
 
-            // Accessors will assume that Type is available.
-            _addMethod = new SynthesizedEventAccessorSymbol(this, isAdder: true);
-            _removeMethod = new SynthesizedEventAccessorSymbol(this, isAdder: false);
+            // Accessors will assume that Type is available. We consider abstract field events to not have a body, but
+            // all other field events do have one.  hasAnyBody here effectively is just proxy for marking if something
+            // is abstract or not. Note: unclear if this should also apply to Extern events.
```

Comment:
>Note: unclear if this should also apply to Extern events. [](http://example.com/codeflow?start=35&length=57)

Yes it should also apply to Extern events as well. Good catch. Please make an adjustment.

Reply:
More specifically: ```hasAnyBody: !IsAbstract && !IsExtern```

Reply:
Done.

Reply:
> Done.

You might want to remove this sentence from the comment though. I think I answered the question. Also, consider revising the comment to not talk about events having body, this code deals with accessors and bodies for accessors.

Reply:
Done!